integrationTests/singleShard/block/executingMiniblocksSc_test.go

```diff
@@ -524,38 +512,37 @@ func runMultipleRoundsOfTheGame(
 			player.Balance = player.Balance.Set(newBalance)
 		}
 
+		// waiting to disseminate transactions
+		time.Sleep(stepDelay)
+
 		for i := 0; i < nrRoundsToPropagateMultiShard; i++ {
-			time.Sleep(time.Second)
 			startTime := time.Now()
 			proposeBlock(nodes, idxProposers, round)
 			elapsedTime := time.Since(startTime)
 			fmt.Printf("Block Created in %s\n", elapsedTime)
 
-			time.Sleep(time.Second)
```

Comment:
Please put this back. There must be some delay between propose and sync (header and body should be broadcast)

Reply:
propose block has a time.sleep inside :).